twitter.css

```diff
@@ -0,0 +1,290 @@
+.center-wrapper {
+  max-width: 960px;
+  padding-left: 16px;
+  padding-right: 16px;
+  margin: 0 auto;
+}
+
+.main-layout {
+  padding: 4px;
+}
+
+/* .main-layout > * {
+  padding: 2px;
+
+  border: 2px dashed lightblue;
+} */
+
+.three-column-grid__expand-two {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+}
+
+.three-row-grid__expand-two {
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+}
+
+.eight-columns {
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+  grid-gap: 8px;
+}
+
+.two-column-expand-space{
+  display: grid;
+  grid-template-columns: 30px 30px auto;
+  margin: 0;
+}
+
+.main-header {
+  width: 275px;
+  height: 100vh;
+}
+
+.responsive-rows {
+  display: grid;
+  grid-auto-rows: repeat(auto-fit, minmax(800px, 1fr));
+}
+
+/* Layout */
+
+.resources-sidebar {
+  width: 275px;
+}
+
+.twitter-logo {
+  width: 55px;
+  height: 45px;
+  margin: 0;
+  padding: 10px;
+}
+
+.side-bar-menu {
+  font-weight: 800;
+  font-size: 20px;
+  margin-right: 20px;
+  padding: 20px;
+  cursor: pointer;
+}
+
+.side-bar-menu h2 {
+  font-weight: 800;
+  font-size: 20px;
+  margin-right: 20px;
+  padding: 20px;
+}
+
+.side-bar-menu :hover {
+  background-color: var(--twitter-background);
+  border-radius: 30px;
+  color: var(--twitter-color);
+  transition: color 100ms ease-out;
+}
+
+.side-bar-menu.active {
+  color: var(--twitter-color);
+}
+
+.side-bar-button__tweet {
+  width: 100%;
+  background-color: var(--twitter-color);
+  color: white;
+  border: none;
+  font-weight: 900;
+  border-radius: 30px;
+  margin-top: 20px;
+  height: 50px;
+}
+
+/* left side-bar */
+
+/* Feed Section */
+a {
+  text-decoration: none;
+
+}
+
+.scroll {
+  overflow: scroll;
+}
+
+.box-md {
+  height: 56px;
+}
+
+.main-box {
+  border: 2px solid var(--twitter-background);
+  margin: 3px;
+  min-height: 56px;
+  position: sticky;
+}
+
+.post-box {
+  border: 2px solid var(--twitter-background);
+  margin: 3px;
+  min-height: 300px;
+  overflow: scroll;
+}
+
+
+.menu-list {
+  list-style-type: none;
+  padding: 20px;
+  margin: 0;
+}
+
+.menu-list a {
+  font-weight: 800;
+  font-size: 20px;
+  margin-right: 20px;
+  /* padding: 20px; */
+  color: black;
+}
+
+.menu-list img {
+   font-weight: 800;
+   font-size: 20px;
+   margin-right: 10px;
+   padding: 30px;
+   padding-bottom: 0px;
+  }
+
+.active {
+  color: var(--twitter-color)
+}
+
+.menu-list  :hover {
+  /* background-color: var(--twitter-background);
+  border-radius: 40px; */
+  color: var(--twitter-color);
+  transition: color 100ms ease-out;
+
+}
+
+.feed-menu-icon {
+  width: 25px;
+  height: auto;
```

Comment:
I think adding `height: auto;` is a bit redundant because that's the initial state of the attribute